src/Compilers/VisualBasic/Portable/SymbolDisplay/CustomSymbolDisplayFormatter.vb

```diff
@@ -124,5 +122,23 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         Public Shared Function ShortNameWithTypeArgsAndContainingTypes(symbol As Symbol) As FormattedSymbol
             Return New FormattedSymbol(symbol, ShortWithTypeArgsAndContainingTypesFormat)
         End Function
+
+        Public Shared Function DefaultErrorFormat(symbol As Symbol) As FormattedSymbol
+            Return New FormattedSymbol(symbol, SymbolDisplayFormat.VisualBasicErrorMessageFormat)
+        End Function
+
+        Public Shared Function DefaultErrorFormatIfSpecialType(arg As Object) As Object
+
+            Dim symbol = TryCast(arg, Symbol)
```

Comment:
yep, you are right, intention was bailing out right away if given arg is not symbol at all (such as FormattedSymbol). but at the end result is same if it is not typesymbol.